### ISSUE TITLE
The demo is broken

### ISSUE THREAD

**Issue by nozof** [tnightingale/Leaflet.widget#5]
The demo is broken http://tnightingale.github.io/Leaflet.widget/demo.html

**Comment by adriennn** [comment]
Yes, if you input custom geoJSON is reverts to the default when clicking on submit query
